articles/healthcare-apis/fhir-oss-powershell-quickstart.md

```diff
@@ -58,4 +58,4 @@ Remove-AzureRmResourceGroup -Name $rg.ResourceGroupName
 In this tutorial, you've deployed the Microsoft Open Source FHIR Server for Azure into your subscription. To learn how to access the FHIR API using Postman, proceed to the Postman tutorial.
  
 >[!div class="nextstepaction"]
->[Access FHIR API using Postman](access-fhir-postman-tutorial.md)
```

Comment:
What is the change here on line 61?

Reply:
I verified there is no change on line 161.

Reply:
Sorry - line 61. Thanks

Reply:
I only changed line 34, but I made the change via the GitHub website's editor and it flagged line 61 as changed as well despite me having changed nothing. Since the change was merged, I assume this was okay. Thanks.